From 46d9e5dae91271fdaa5dbcc8d031aa510159bba4 Mon Sep 17 00:00:00 2001
From: Andarwinux <Andarwinux@users.noreply.github.com>
Date: Thu, 1 Feb 2024 00:00:00 +0000
Subject: [PATCH] vulkan-header: reset to 1.3.276

FFmpeg VK_MESA_video_decode_av1 structure conflicted with VK_KHR_video_decode_av1 structure in 1.3.277 cause build to fail, so temporarily reset to ea45703 until FFmpeg fixes the problem.
---
 packages/vulkan-header.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/vulkan-header.cmake b/packages/vulkan-header.cmake
index bf18b3644..4557471df 100644
--- a/packages/vulkan-header.cmake
+++ b/packages/vulkan-header.cmake
@@ -5,6 +5,7 @@ ExternalProject_Add(vulkan-header
     UPDATE_COMMAND ""
     GIT_REMOTE_NAME origin
     GIT_TAG main
+    GIT_RESET ea45703effcb01df0856628286f8a890dd313ecd
     CONFIGURE_COMMAND ${EXEC} CONF=1 cmake -H<SOURCE_DIR> -B<BINARY_DIR>
         -G Ninja
         -DCMAKE_BUILD_TYPE=Release
